src/Sulu/Bundle/AutomationBundle/Tests/Unit/Tasks/Manager/TaskManagerTest.php

```diff
@@ -112,6 +112,10 @@ function (TaskEvent $event) use ($task) {
                     return $task == $event->getTask();
                 }
             )
-        )->willReturnArgument(1);
+        )->will(
```

Comment:
@danrot prophecy ^1.4 provides this. so adding the dependency wouldnt be a problem. but i dont see the big disadvantage of this code-snippet.